packages/iris-grid/src/IrisGridUtils.ts

```diff
@@ -1305,9 +1297,9 @@ class IrisGridUtils {
    * @returns The columns selected in the range
    */
   static columnsFromRanges(
-    ranges: GridRange[],
-    allColumns: Column[]
-  ): Column[] {
+    ranges: readonly GridRange[],
+    allColumns: readonly Column[]
+  ): readonly Column[] {
```

Comment:
Probably don't need to return readonly. Although I see this one might be b/c you return the original array if the start/end is null. Maybe it should return a clone of that array instead?

Reply:
Yea I think it should clone the array instead and be consistent. Similar to the `subtractFromRanges` method in `GridRange`